Makefile

```diff
@@ -111,13 +111,16 @@ deploy: ## 🚀 Deploy to k8s context defined by $KUBECTX (default: kind-rig)
 		--set image.tag=$(TAG) \
 		--set mongodb.enabled=true \
 		--set rig.telemetry.enabled=false \
+		--set rig.cluster.dev_registry.enabled=true \
```

Comment:
Let's remove `enabled`, and just let the host below imply if it's enabled or not